FluentUI/src/main/res/values/colors.xml

```diff
@@ -66,6 +66,10 @@
         <item>#567C73</item>
         <item>#69797E</item>
     </array>
+    <color name="fluentui_avatar_ring_background">@color/fluentui_white</color>
+    <color name="fluentui_avatar_border_background">@color/fluentui_gray_200</color>
```

Comment:
customized as avatar bg is customized , it may not always fit

Reply:
this is non-customisable